src/js/components/Calendar/stories/HeaderLevel.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+import { Box, Calendar } from 'grommet';
+
+export const HeaderLevel = () => {
+  const [date, setDate] = useState();
+
+  const onSelect = (nextDate) => {
+    setDate(nextDate !== date ? nextDate : undefined);
+  };
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box>
+      <Box align="center" pad="large">
+        <Calendar
+          date={date}
+          daysOfWeek
+          onSelect={onSelect}
+          level={2}
+          bounds={['2020-09-08', '2025-12-13']}
+        />
+      </Box>
+    </Box>
+    // </Grommet>
+  );
+};
+
+export default {
+  title: 'Visualizations/Calendar/HeaderLevel',
```

Comment:
Can we change the story name to 'Heading level' with a space and lowercase 'l' to match the other stories?

Reply:
done